app/styles/libs/_helpers.scss

```diff
@@ -131,10 +131,28 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
   width: fit-content;
 }
 
-.text-white {
```

Comment:
Why

Reply:
Will add in the latest PR sir, Sorry for inconvenience

Reply:
Sir due to addition of css in the middle this styling is shifted to line 145

Reply:
Sir I did the changes , Please review it again.

Reply:
This class is still removed

Reply:
Sir, I am not able to understand how this class is removed,  as you can see in the screenshot given,  It just shifed to line 145 from 133 ,not removed.
![Screenshot (893)](https://user-images.githubusercontent.com/63748249/106031762-a667b300-60f5-11eb-8ec2-815d1c7f70b6.png)

Reply:
Revert it to where it was

Reply:
Done sir

Reply:
@iamareebjamal  and @mariobehling  Sir Please review this PR.